### ISSUE TITLE
Better focus order and keyboard navigation in actions, search and stage

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#2233]
We should improve the focus order when tabbing and general keyboard navigation ability in OCMS .  Any updates should be tested in Chrome, IE and Firefox. 

Some areas to look at:

- [ ] Send Email Action - Chrome Focus order goes: To, Templates button (if configured), Upload attachments button, browser nav bar SSL icon, nav bar location, Subject, email body, email note, cc, bcc
- [ ] Search - the trac selector and search type dropdowns  and "more fields" button are never accessible via `[Tab]`.
- [ ] Stage folder actions - never accessible via a keyboard
- [ ] Cannot click an item in table view via a keyboard
- Other areas?  I'm sure there are more we can do better with.
